setup/vmSetupAzSecPack.sh

```diff
@@ -0,0 +1,103 @@
+# Install Azure CLI
```

Comment:
We shall call this script from the above setup script to set up everything in one shot